Update README.md (#1)
gbaranski/gbaranski: README.md

### What am I familiar with ?

> My experience is mostly with networking and system programming, however some projects required me to also code some frontend app or do something on microcontrollers like STM32 or ESP8266. And I like experiments, trying new things, that gives me a lot of fun.

Reviewer: ```suggestion
My experience is mostly with dev/ops, networking and system programming. Also, I'm familiar with low-level and frontend development since some projects involved microcontrollers (STM32 or ESP8266) and required me to build a user interface. 
I enjoy experimenting and trying new things.
```

> I'm pretty fluent with Rust, moreover languages like Go or Typescript aren't foreign to me.

Reviewer: ```suggestion
I'm fluent with Rust, less with Go and Typescript.
```

Author: Don’t you think that using "less with ..." makes it sound as if I’m unfamiliar with Go & TS.
What do you think about “I’m fluent with Rust, and familiar with Go and Typescript too.”

Reviewer: Ok, but I would get rid of "... too" — it's redundant.

> It's essential for me to have a large toolbox, so I can pick the best tool for the problem, and since I was working as a one-man army for most of my projects, I needed a lot of different tools.

Reviewer: ```suggestion
Before I get my hands dirty and jump to programming, I like to research and find the most suitable tools, frameworks, libraries, languages; and learn them if needed.
```

---

> I'm currently attending to a high school in Poland, with specialization in mechatronics, although I don't assosiate my future with this profession.

Reviewer: ```suggestion
I'm currently attending a high school in Poland with a specialization in mechatronics. But, what fascinates me the most is software development, and therefore my future is in programming.
```

> - Fun fact: Back when I was 13, I created a autoclicker for Minecraft using AutoHotkey, but after the project was done, I made a 2 year break from programming.

Reviewer: ```suggestion
> Fun fact: Back when I was 13, I created an auto clicker for Minecraft using AutoHotkey. When the project was done, I made a 2-year break from programming.
```

### What am I familiar with ?

My experience is mostly with networking and system programming. Also, I'm familiar with low-level and frontend development since some projects involved microcontrollers (STM32 or ESP8266) and required me to build a user interface. 
I enjoy experimenting and trying new things.

> I'm fluent with Rust, less with Go and Typescript.

Author: ```suggestion
I'm fluent with Rust, and familiar with Go and Typescript too.
```
